### PR TITLE
remove GA feature-gate APISelfSubjectReview

### PR DESCRIPTION
**Pull request by carlory**
ref https://github.com/kubernetes/kubernetes/issues/120315
feature-gate was removed by https://github.com/kubernetes/kubernetes/pull/122032

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates/index.md
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/index.md:1]: Hi. You also need to remove this line:
```markdown
- {{< feature-gate-description name="APISelfSubjectReview" >}}
```

**Review by sftim — CHANGES_REQUESTED** [review]
Thanks

**Comment by sftim** [comment]
/lgtm cancel

Once more change needed.

File: content/en/docs/reference/command-line-tools-reference/feature-gates/index.md
Comment (carlory) [reply content/en/docs/reference/command-line-tools-reference/feature-gates/index.md:1]: @sftim thank you. I forget it. 

It's removed now. please review again.

**Review by sftim — COMMENTED** [review]
Thanks

/lgtm
/approve

**Pull request merged**